connexion/security/async_security_handler_factory.py

```diff
@@ -3,8 +3,6 @@
 import functools
 import logging
 
-import aiohttp
```

Comment:
@kigawas can you PR this separately to the main connexion branch so that we don't mess with files we are not going to change?

Reply:
Since the sanic dependencies do not include aiohttp, it's better to remove this to avoid potential import error

Reply:
https://github.com/zalando/connexion/pull/1320

pr proposed

Reply:
@kigawas I understand, but let's push on #1320 and remove this change from this PR so we don't have to rebase :)

Reply:
No worries, I'll update later